source/includes/_endpoint-dataset.md

```diff
@@ -237,3 +237,27 @@ description | string | "" | A longer description of the dataset
 #### DELETE
 
 `DELETE /datasets/{dataset_id}/`
+
+### Summary
+
+`/datasets/{id}/summary/{?filter_syntax,filter_url,filter}`
```

Comment:
Don't forget to remove filter_syntax and filter_url once you delete those from backend-http.